libflux/flux-core/src/ast/mod.rs

```diff
@@ -102,6 +102,16 @@ impl SourceLocation {
     }
 }
 
+#[cfg(feature = "lsp")]
+impl Into<lsp_types::Range> for SourceLocation {
```

Comment:
You can implement `From<SourceLocation> for lsp_types::Range` instead and you get `Into` for free, same for the existing `Position` impl.

Reply:
Does it not work both ways? If I implement `Into`, do I not get `From` for free? If not, we should definitely change this.

Reply:
No, there is a blanket `impl<T> Into<T> for T where T: From<T>` but there can't be one the other way around as it would overlap.

Reply:
Excellent, thanks for catching that. I'd rather implement `From` across the board. Not sure what I was thinking with the original `Into`, as I have never implemented that before. Updated both `Into` implementations into `From` implementations.